<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntersectionDialogForm</class>
 <widget class="QDialog" name="IntersectionDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intersection Operation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="m_datasourceTypeTitleLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Intersection Operation...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="m_imgLabel">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>Imagem</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="m_filterLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select the input layers</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QTreeView" name="m_layerTreeView"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="m_advancedGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Advanced</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QRadioButton" name="m_wholeMemRadioButton">
                <property name="toolTip">
                 <string>operation with all layers in memory</string>
                </property>
                <property name="text">
                 <string>Whole operation in memory</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="m_lowMemRadioButton">
                <property name="toolTip">
                 <string>operation without memory use</string>
                </property>
                <property name="text">
                 <string>Low memory use</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="m_partiallyMemRadioButton">
                <property name="toolTip">
                 <string>operation with only one layer in memory</string>
                </property>
                <property name="text">
                 <string>Partially operation in memory</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Output</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>New Layer Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Target Repository:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="toolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="m_datasourcesToolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>BD</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QPushButton" name="m_helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_okPushButton">
         <property name="text">
          <string>&amp;Ok</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButtonm_cancelP">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
